src/commands/config/create.ts

```diff
@@ -0,0 +1,191 @@
+import { OpenAttestationDocument } from "@govtechsg/open-attestation";
+import fs from "fs";
+import { error, info, success } from "signale";
+import { Argv } from "yargs";
+import { deployDocumentStore } from "../../implementations/deploy/document-store";
+import { deployTokenRegistry } from "../../implementations/deploy/token-registry";
+import { readFile } from "../../implementations/utils/disk";
+import { create as createWallet } from "../../implementations/wallet/create";
+import { getLogger } from "../../logger";
+import { highlight } from "../../utils";
+import { handler as createTemporaryDns } from "../dns/txt-record/create";
+import { CreateConfigCommand } from "./config.type";
+
+interface ConfigFile {
+  wallet: string;
+  forms: Form[];
+}
+
+interface Form {
+  type: "VERIFIABLE_DOCUMENT" | "TRANSFERABLE_RECORD";
+  defaults: {
+    issuers: OpenAttestationDocument[];
+  };
+}
+
+const { trace } = getLogger("config:create");
+
+export const command = "create [options]";
+
+export const describe = "Create a config file";
+
+export const builder = (yargs: Argv): Argv =>
+  yargs
+    .option("output-dir", {
+      alias: "od",
+      description: "Write output to a directory",
+      type: "string",
+      demandOption: true,
+    })
+    .option("encrypted-wallet-path", {
+      type: "string",
+      description: "Path to file containing wallet.json",
+      normalize: true,
+    })
+    .option("config-template-path", {
+      type: "string",
+      description: "Path to file containing config template",
+      normalize: true,
+    })
+    .option("config-type", {
```

Comment:
let's add conflicts for config-template-path and config-type: https://yargs.js.org/docs/#api-reference-conflictsx-y

Reply:
ok